x-pack/plugins/apm/public/tutorial/config_agent/get_policy_options.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { i18n } from '@kbn/i18n';
+import { APIResponseType } from './';
+
+const POLICY_ELASTIC_AGENT_ON_CLOUD = 'policy-elastic-agent-on-cloud';
+
+const DEFAULT_STANDALONE_CONFIG_LABEL = i18n.translate(
+  'xpack.apm.tutorial.agent_config.defaultStandaloneConfig',
+  { defaultMessage: 'Default Standalone configuration' }
+);
+
+export type PolicyOption = ReturnType<typeof getPolicyOptions>[0];
+
+export function getPolicyOptions({
+  isCloudEnabled,
+  data,
+}: {
+  isCloudEnabled: boolean;
+  data: APIResponseType;
+}) {
+  const isCloudVisible = !!(
+    isCloudEnabled && data.cloudStandaloneSetup?.apmServerUrl
+  );
+
```

Comment:
Probably out-of-scope, but I wonder if we could leverage the flags from `GET /api/apm/fleet/migration_check` to pre-select options https://github.com/elastic/kibana/pull/102682/files#diff-9c2335d860c78c5a6494892a1deda7dbea6206f7eb8388a31c2a11b2b4033cd9R117-R120

Reply:
I don't think so, `isCloudEnabled` doesn't come from the API but it's passed as a property, and `data.cloudStandaloneSetup` is the URL and secretToken to access the cloud env.